src/ui/utils.c

```diff
@@ -103,4 +107,13 @@ bool address_from_pubkey(const uint8_t public_key[static 64], char* out, size_t
     // step 3
     script_hash_to_address(out, out_len, script_hash);
     return true;
-}
\ No newline at end of file
+}
+
+#ifdef HAVE_NBGL
+void releaseContext(void) {
```

Comment:
Not sure you still need this one if you use only the higher level APIs of the SDK :thinking:

Reply:
Removed